pkgs/tools/misc/gtklp/default.nix

```
@@ -0,0 +1,55 @@
+{ stdenv, fetchurl
+, autoconf, automake, libtool, pkgconfig, fetchpatch
+, gtk2, glib, cups
+, gettext, openssl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gtklp";
+  version = "1.3.4";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/${pname}/${pname}-${version}.src.tar.gz";
+    sha256 = "1arvnnvar22ipgnzqqq8xh0kkwyf71q2sfsf0crajpsr8a8601xy";
+  };
+
+  nativeBuildInputs = [
+    pkgconfig
+  ];
+
+  buildInputs = [
+    autoconf
+    automake
+    cups
+    gettext
+    glib
+    gtk2
+    libtool
+    openssl
+  ];
+
+  patches = [ ../gtklp/formatsec.patch ];
```

Add a comment please. and use:

```suggestion
  patches = [ ./formatsec.patch ];
```